app/models/manageiq/providers/cisco_intersight/inventory/parser/physical_infra_manager.rb

```diff
@@ -11,7 +11,7 @@ def parse
     # Methods that are executed during refresh call
 
     def physical_servers
-      # Parses physical servers and its details
+      # Parsing active/undecomissioned servers:
```

Comment:
Don't believe `undecomissioned` is a word, might just `Parsing active/ servers` work or `active/non-decommissioned` ?

Reply:
True. Just writing "parsing active servers" with the next commit.